applications/serial_lte_modem/doc/MQTT_AT_commands.rst

```diff
@@ -93,14 +94,10 @@ Response syntax
 
 ::
 
-   #XMQTTCON: <cid>,<username>,<password>,<url>,<port>[,<sec_tag>]]
+   #XMQTTCON: <client_id>,<url>,<port>[,<sec_tag>]]
 
-* The ``<cid>`` value is a string.
+* The ``<client_id>`` value is a string.
   It indicates the MQTT client ID.
-* The ``<username>`` value is a string.
```

Comment:
It doesn't seem that MQTT username/password processing was removed from the code.

Reply:
It's for #XMQTTCON?, that normally logon credentials are not shown.

Reply:
Ah, right, fine then